### PR TITLE
added env. for remoteConfig

### PR DESCRIPTION
**Pull request by himanshu-2764**

### CODE REVIEW

File: src/modules/env/.env.dev [hunk 1]
```diff
@@ -20,6 +20,8 @@ build='{
 build__Client=Syngenta
 build__Project= 
 build__Stage=dev
+remoteConfig__baseUrl=https://cropwise-api-node-css-config-dev.azurewebsites.net
```
Comment (RambhauWaykar) [inline src/modules/env/.env.dev:23]: change url to https://cropwise-api-node-css-config-staging.azurewebsites.net

File: src/modules/env/.env.qa [hunk 2]
```diff
@@ -18,6 +18,8 @@ build='{
 build__Client=Syngenta
 build__Project= 
 build__Stage=qa
+remoteConfig__baseUrl=https://cropwise-api-node-css-config-staging.azurewebsites.net
+remoteConfig__environment=uat
```
Comment (RambhauWaykar) [inline src/modules/env/.env.qa:22]: change environment to qa

File: src/modules/env/.env.staging [hunk 3]
```diff
@@ -18,6 +18,8 @@ build='{
 build__Client=Syngenta
 build__Project= 
 build__Stage=uat
+remoteConfig__baseUrl=https://cropwise-api-node-css-config-staging.azurewebsites.net
+remoteConfig__environment=staging
```
Comment (RambhauWaykar) [inline src/modules/env/.env.staging:22]: change environment to uat

**Review by RambhauWaykar — CHANGES_REQUESTED** [review]

File: src/modules/env/.env.uat [hunk 4]
```diff
@@ -18,6 +18,8 @@ build='{
 build__Client=Syngenta
 build__Project= 
 build__Stage=uat
+remoteConfig__baseUrl=https://cropwise-api-node-css-config-staging.azurewebsites.net
```
Comment (vansham69420) [inline src/modules/env/.env.uat:21]: remoteConfig=`{}`

**Review by RambhauWaykar — APPROVED** [review]

**Review by vansham69420 — APPROVED** [review]

**Review by prashantSyngenta — APPROVED** [review]

**Pull request merged**